.github/workflows/python-build-standalone.yml

```diff
@@ -21,10 +21,10 @@ jobs:
       - name: Sync Python Releases
         run: |
           uv run --isolated -- fetch-download-metadata.py
-          uv run --isolated --with chevron-blue -- template-download-metadata.py
+          uv run --isolated -- template-download-metadata.py
         working-directory: ./crates/uv-python
         env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Huh, TIL both are supported: https://cli.github.com/manual/gh_help_environment

I have a preference for reading `GITHUB_TOKEN` in the script but only a small one.